pkg/terminal/terminal.go

```diff
@@ -84,30 +85,41 @@ func New(client service.Client, conf *config.Config) *Term {
 		conf = &config.Config{}
 	}
 
-	var w io.Writer
-
-	dumb := strings.ToLower(os.Getenv("TERM")) == "dumb"
-	if dumb {
-		w = os.Stdout
-	} else {
-		w = getColorableWriter()
-	}
-
-	if (conf.SourceListLineColor > ansiWhite &&
-		conf.SourceListLineColor < ansiBrBlack) ||
-		conf.SourceListLineColor < ansiBlack ||
-		conf.SourceListLineColor > ansiBrWhite {
-		conf.SourceListLineColor = ansiBlue
-	}
-
 	t := &Term{
 		client: client,
 		conf:   conf,
 		prompt: "(dlv) ",
 		line:   liner.NewLiner(),
 		cmds:   cmds,
-		dumb:   dumb,
-		stdout: w,
+		stdout: os.Stdout,
+	}
+
+	if strings.ToLower(os.Getenv("TERM")) != "dumb" {
+		t.stdout = getColorableWriter()
+		t.colorEscapes = make(map[colorize.Style]string)
+		t.colorEscapes[colorize.NormalStyle] = terminalResetEscapeCode
+		wd := func(s string, defaultCode int) string {
+			if s == "" {
+				return fmt.Sprintf(terminalHighlightEscapeCode, defaultCode)
+			}
+			return s
+		}
+		t.colorEscapes[colorize.KeywordStyle] = conf.SourceListKeywordColor
+		t.colorEscapes[colorize.StringStyle] = wd(conf.SourceListStringColor, ansiBrGreen)
+		t.colorEscapes[colorize.NumberStyle] = conf.SourceListNumberColor
+		t.colorEscapes[colorize.CommentStyle] = wd(conf.SourceListCommentColor, ansiBrMagenta)
+		t.colorEscapes[colorize.ArrowStyle] = wd(conf.SourceListArrowColor, ansiBrYellow)
```

Comment:
On a light-theme terminal, the bright yellow arrow is barely visible.
Additionally, the bright green for strings from line 108 is too bright.
This is on Konsole's "Black on White" theme, which is the default on OpenSUSE. It also looks the same on default xterm.
We can't please everyone, but I think it's possible to find default colors that work for the most common default colorschemes out there (white-on-black, black-on-white and maybe the Ubuntu one with purple background).